examples/06_scope_functions/03_with.md

```diff
@@ -1,6 +1,8 @@
+
+
 # with
 
-*with* function lets you access members of an object in a short and concise way. Inside the `with` block you can refer to object members without specifying its name as a prefix. 
+`with` is a non-extension function that can be used to access members of its argument in a concise way: you can omit the instance when referring to its members. 
```

Comment:
`with` is a non-extension function that can access members of its argument concisely: you can omit the instance when referring to its members.